clients/js/src/index.ts

```diff
@@ -15,7 +15,12 @@ export class CacReader {
     defaultConfig: IObject;
 
     static {
-        jsonLogic.add_operation(">>", compareSemanticIsGreater);
+        jsonLogic.add_operation("match", matchRegex);
+        jsonLogic.add_operation("semver_eq", compareSemanticIsEqual);
+        jsonLogic.add_operation("semver_gt", compareSemanticIsGreater);
+        jsonLogic.add_operation("semver_gteq", compareSemanticIsGreaterEqual);
```

Comment:
@ayushjain17 should we add one for between also ?

Reply:
@pratikmishra356 less than supports between operation as well, the behaviour of these version comparison operators are same as that of the normal comparison operators - wrt the number of operators